app/gui/src/controller/graph/executed.rs

```diff
@@ -416,6 +416,7 @@ pub mod tests {
         // Setup the controller.
         let mut fixture = crate::test::mock::Unified::new().fixture();
         let Fixture { executed_graph, execution, executor, .. } = &mut fixture;
+        executor.run_until_stalled();
```

Comment:
If it is required at this point, shan't it be called always when constructing the fixture?